.plzconfig

```diff
@@ -1,3 +1,7 @@
+[please]
+location = ~/.please
+version = 15.0.0-beta9
```

Comment:
maybe `version = >=15.0.0-beta9`? then we won't get nagged when we release newer ones...